lua/femaco/edit.lua

```diff
@@ -188,17 +189,25 @@ M.edit_code_block = function()
   local float_bufnr = vim.fn.bufnr()
   vim.api.nvim_create_autocmd({'BufWritePost', 'WinClosed'}, {
     buffer = 0,
-    callback = function()
+    callback = function(event)
       local lines = vim.api.nvim_buf_get_lines(float_bufnr, 0, -1, true)
 
-      if tbl_equal(match_lines, lines) then return end
+      if tbl_equal(match_lines, lines) then
+        if event.event == 'WinClosed' then
```

Comment:
could this be moved before checking if nothing changed to avoid having this clause both here and below?

Reply:
I guess nothing relies on the file content in this callback, so that makes sense. Good call!